core/jazz_asset-event-handler/test/test.js

```diff
@@ -508,29 +514,86 @@ describe('jazz asset handler tests: ', function () {
       })
     });
 
-    it("should call processEvents",()=>{
-      processEventsStub = sinon.stub(index, "processEvents").resolves(result);
-      index.handler(event, context, (error, records)=>{
-        sinon.assert.calledOnce(processEventsStub);
-        processEventsStub.restore();
-      })
+    it('index should resolve for not interested events', function () {
+      var callback = (err, responseObj) => {
+        if (err) {
+          return err;
+        }
+        else {
+          return JSON.stringify(responseObj);
+        }
+      };
+
+      tokenResponseObj.statusCode = 400;
+
+      var authStub = sinon.stub(rp, 'Request').returns(Promise.resolve(tokenResponseObj));
+      var reqStub = sinon.stub(request, "Request").callsFake(obj => {
+        return obj.callback(null, responseObject, responseObject.body);
+      });
+      var message = 'User is not authorized to access this service';
+      index.handler(event, context, (err, res) => {
+        if (err) {
```

Comment:
Please don't do if else in the unit test. Unit test should be set up to either test for resolve or reject. Please create 2 unit tests here.

Reply:
Updated this test,